### ISSUE TITLE
A question about Pano's data folder

### ISSUE THREAD

**Issue by puelp** [kawaiiDango/pano-scrobbler#378]
- [ O ] My question is not listed in the FAQ at https://telegra.ph/Pano-Scrobbler-FAQ-05-30

This is related to https://github.com/kawaiiDango/pScrobbler/issues/377

I thought maybe there would be some useful logs in the Pano data folder, so I used adb backup to extract the data folder.

That resulted in an .ab file that was only like 500 bytes big, too small to hold any useful scrobble data.

But I figured that I would come here and ask about it, just in case I did something wrong.

1. A 500 byte .ab backup file is normal for Pano, yes? It's not just me screwing something up?
2. There's no secret folder elsewhere in which Pano keeps the scrobble logs, right? The 500 byte .ab file is all there is to Pano data?

**Comment by kawaiiDango** [comment]
This is weird. Maybe my backup_rules.xml does not allow everything to be backed up by adb.

The scrobble logs are printed to logcat. If you use Help -> Bug report, the logcat gets dumped into a log.txt which gets stored in data folder and gets attached to the email

**Comment by puelp** [comment]
> This is weird. Maybe my backup_rules.xml does not allow everything to be backed up by adb.

I looked it up, and found this: https://stackoverflow.com/questions/74387730/back-up-android-with-adb-backup

_adb backup fails on recent apps that have targetSDK level to 31 or higher - only exception: the app is a debuggable app (debuggable="true" in AndroidManifest.xml) but this is never the case for apps from Play Store or third party download sites)._

Does that help or no?

> The scrobble logs are printed to logcat. If you use Help -> Bug report, the logcat gets dumped into a log.txt which gets stored in data folder and gets attached to the email

OK I did it but the log only has stuff from the past few minutes/hour(s).

Is there a way to go back further than that?
